[Music] hello everyone I'm ready I'm a graduate student at the Earth Life Science Institute um in our study we have tried to understand scaling of protein function across the tree of life and the mechanisms that might have led to the species diversity on the tree of life right here so we study scaling using power loss and power laws are found everywhere if we consider the number of Web Hits on a web page in a given period of time or the earthquake magnitude in an area over a given period of time many natural and man made processes follow power loss [Music] so how can we help um help us how can this help us understand some Concepts in biology for that let's consider a Lego set and consider the unique pieces in relation to the total pieces in the Lego set when we plot this on a log log scale we see the larger Lego sets use more Unique Piece types but they progressively go on using lesser additional piece types so they're becoming more efficient which means the larger sets are using uh the same pieces the smaller sets are using but in more efficient and more complex ways so what we're observing in these plots is a scaling relationship and when we observe a scaling relationship we could say that maybe there's a set of rules that sort of governing the way something is working so you as a power law equation one quantity varying as a power law of the other and when we plot this on a log log scale we get a straight line with the slope Alpha so previous Studies have shown that genes in a specific functional category scale as a power law of the total number of genes in a genome so for example transcription regulation is almost quadratically scaling which means if the genome doubles in size the genes in this specific category are going to quadruple so we have tried to include an expanded taxonomy in our study and for that we use the eggnog database so after power of fitting we saw different Trends in our data for the smaller and the larger genomes so we carried out piecewise regression to give Justice to the different patterns and scaling observed in the plots so for example we wanted to capture the slope variability for the smaller and the larger genome sizes so for example in category M which is cell wall and cell membrane proteins um so the x axis has the total protein annotations and the y axis has the category annotations for that specific category so for the smaller genomes we can see the proteins are scaling fast which means as the genome size is increasing they are incorporating um more and more proteins faster than the larger genomes because after a statistically detected breakpoint the scaling slows down and we can see a similar Trend in category tree category T so we saw this trend in most of the breakpoints that that were supported in bacteria interestingly we observed an opposite Trend in archaea the scaling is slow in the start but fastens up after the statistically detected breakpoint so a lot of categories were common between archaea and bacteria um but there were some categories that were exclusively present either in archaeon bacteria um and it's also interesting to observe these differences in scaling pattern before and after the break point in Archaea and bacteria so we thought maybe these differences in scaling patterns were caused by phyla specific scaling so we broke these domains down into specific phyla and found great variation in all the phyla for all the categories so for example we have category H here which is coenzyme transport proteins um which also happens to be the most variable across all the phyla so we thought maybe this file a specific scaling is causing the positioning of the breakpoints that we observed previously um so I place these breakpoints on the total protein annotations to see if there's any specific pattern but we can see these individual phyla are spanning the breakpoints and there is no specific preference of for the break the file as to be present either on either sides of the breakpoints so maybe taxonomy is not causing the positioning of these breakpoints and maybe there are some other factors like physiological or environmental factors that are causing these fake points so we were also interested in these groups CPR and d pan uh so these groups have extremely small genomes and they lack um major metabolic pathways uh so we compared them with um eukaryotes unicellular eukaryotes and Asgard alkia so for some categories we can see the scaling is very similar for category o but for some categories the scaling is very different like in category C which goes on to show there are different ways in which an organism can adapt while growing in their genome sizes um I've just discussed a few key results in my uh talk so if you want to discuss I would be interested please come by and stop at panel two for the poster thank you [Music] [Applause] [Music] thank you [Music]